README.md

```diff
@@ -22,6 +22,8 @@ After installation the Azure Resource Manager Idem Provider execution and state
 
 The provider authenticates with a service principal, so all state and execution modules require that a dictionary
 populated with the data shown below be passed to them.
+
+Lets call the file myawesomecreds.yml
 ```
 {% set profile = {
```

Comment:
This Jinja section needs to be caveated with something like "we should do it this way anymore" or removed altogether. I'm leaning toward the latter.